api/layer/multipart_upload.go

```diff
@@ -500,18 +505,29 @@ func (n *layer) getUploadParts(ctx context.Context, p *UploadInfoParams) (map[in
 			continue
 		}
 		info := objInfoFromMeta(p.Bkt, meta)
+		// skip objects which are completed by "complete-multipart-upload" because they have "s3-Upload-Id" attribute
+		if !isSystem(info) {
+			continue
+		}
```

Comment:
Could you explain this? As I understand all parts are system now so they will all be skipped.

Reply:
But we skip common files not system. We check **!** isSystem()